### DISCUSSION TITLE
How to set (ff-interface) UI Language

### DISCUSSION THREAD

**Discussion by native-engine** [fontforge/fontforge#4800] [Q&A]
How do I switch the program language?
It's too bad that this option is not available in the settings!

**Comment by ctrlcctrlv** [comment]
It's determined by the operating system locale.

**Reply by ChiefMikeK** [reply]
Searching github.com is difficult @ctrlcctrlv copy-paste
- Please Change the title to something such as.

## How to set (ff-interface) UI Language
- https://github.com/fontforge/designwithfontforge.com/issues/176

**Reply by ctrlcctrlv** [reply]
Done
